test/xml/xml_api_test.cc

```diff
@@ -64,8 +64,38 @@ TEST_F(LoadXmlTest, InvalidXmlFailsToLoad) {
     mj_deleteModel(model);
   }
 }
+
+TEST_F(LoadXmlTest, MultipleBodies) {
+  static constexpr char xml[] = R"(
+  <mujoco>
+    <worldbody>
+      <body>
+        <joint/>
+        <geom size="1"/>
+      </body>
+      <body>
+        <joint/>
+        <geom size="0.5"/>
+      </body>
+    </worldbody>
+  </mujoco>
+  )";
+  std::array<char, 1000> error;
+  mjModel* model = LoadModelFromString(xml, error.data(), error.size());
+    
+  ASSERT_THAT(model, NotNull()) << "Failed to load model: " << error.data();
+  EXPECT_EQ(model->nbody, 3);
+    
+  mjData* data = mj_makeData(model);
+  EXPECT_THAT(data, NotNull());
+  mj_step(model, data);
+  mj_deleteData(data);
+  mj_deleteModel(model);
+}
+
 // TODO(nimrod): Add more tests for mj_loadXML.
```

Comment:
Do you mind deleting this line?

This TODO was written at the point where we literally only had one test in all of MuJoCo.
mj_loadXML is now very well covered by many tests.

This new test won't hurt, but I don't think we need more basic tests of this sort.

Reply:
Oh, I understand. I will remove that.